python/pyspark/sql/tests/test_functions.py

```diff
@@ -333,6 +333,14 @@ def test_rand_functions(self):
         rndn2 = df.select("key", F.randn(0)).collect()
         self.assertEqual(sorted(rndn1), sorted(rndn2))
 
+    def test_try_parse_url(self):
```

Comment:
Could we also add a case here that returns null? This is to make sure python works as well with invalid urls.